holoviews/plotting/bokeh/graphs.py

```diff
@@ -101,13 +111,31 @@ def _get_edge_colors(self, element, ranges, edge_data, edge_mapping, style):
         mapper = self._get_colormapper(cdim, element, ranges, edge_style,
                                        factors, colors, 'edge_colormapper')
         transform = {'field': field, 'transform': mapper}
-        edge_mapping['edge_line_color'] = transform
-        edge_mapping['edge_nonselection_line_color'] = transform
-        edge_mapping['edge_selection_line_color'] = transform
+        color_type = 'fill_color' if self.filled else 'line_color'
+        edge_mapping['edge_'+color_type] = transform
+        edge_mapping['edge_nonselection_'+color_type] = transform
+        edge_mapping['edge_selection_'+color_type] = transform
 
 
-    def get_data(self, element, ranges, style):
+    def _get_edge_paths(self, element):
+        path_data = {}
         xidx, yidx = (1, 0) if self.invert_axes else (0, 1)
+        if element._edgepaths is not None:
+            edges = element._split_edgepaths.split(datatype='array', dimensions=element.edgepaths.kdims)
+            if len(edges) == len(element):
+                path_data['xs'] = [path[:, xidx] for path in edges]
+                path_data['ys'] = [path[:, yidx] for path in edges]
+            else:
+                self.warning('Graph edge paths do not match the number of abstract edges '
+                             'and will be skipped')
```

Comment:
Definitely a warning and not an error condition?

Reply:
Would be happy to make it an error, this has been part of the Graph implementation since first merging it and I've not run into it.